.github/actions/setup-environment/action.yml

```diff
@@ -299,9 +299,9 @@ runs:
             echo "crn-auth0-domain=pr-asap-hub.us.auth0.com" >> $GITHUB_OUTPUT
           fi
 
-          CRN_CREATE_ANALYTICS_ALGOLIA_INDEX=${{ contains(github.event.pull_request.labels.*.name, 'crn-create-analytics-algolia-index') || inputs.crn-create-env }}
-          CRN_CREATE_CONTENTFUL_PR_ENV=${{ contains(github.event.pull_request.labels.*.name, 'crn-create-environment') || inputs.crn-create-env }}
-          GP2_CREATE_CONTENTFUL_PR_ENV=${{ contains(github.event.pull_request.labels.*.name, 'gp2-create-environment') || inputs.gp2-create-env }}
+          CRN_CREATE_ANALYTICS_ALGOLIA_INDEX=${{ contains(github.event.pull_request.labels.*.name, 'crn-create-analytics-algolia-index') || inputs.crn-create-env || false }}
```

Comment:
same thing as https://github.com/yldio/asap-hub/pull/4404, just adding it here as well so the PR is not blocked